### PR TITLE
Add new option as --nowrap, support colored message in balloon

### PR DESCRIPTION
**Pull request by sters**
Current:

image

My PR:

image

### CODE REVIEW

File: rainbow.go [hunk 1]
```diff
@@ -68,3 +68,34 @@ func rgb(i float64) int {
 	blue := int(6*((math.Sin(freq*i+4*m)*127+128)/256)) * 1
 	return 16 + red + green + blue
 }
+
+// UnANSI color code syntax
+func UnANSI(str string) string {
```
Comment (sters) [inline rainbow.go]: My Opinion, This is a temporary solution.
Each char have to structure of split text and color.

**Comment by Code-Hex** [comment]
mmm, Actually, I don't want to add the `no-wrap` option. So, I will fix `go-wordwrap` package first, then please fix again this PR?

**Comment by sters** [comment]
Also, I'm trying this package.
https://github.com/sters/colorune

**Comment by Code-Hex** [comment]
@sters I have bumped a new version of the go-wordwrap for ignoring ANSI colors (https://github.com/Code-Hex/go-wordwrap/tree/v1.0.0)
Please try this one!!

**Comment by sters** [comment]
I only revert no-wrap option. It looks too difficult to Control Sequence lol

image

**Comment by sters** [comment]
Conflicted my PR, see you later.

**Pull request closed without merging**